apollo-router/src/plugins/telemetry/config_new/selectors.rs

```diff
@@ -804,6 +815,26 @@ impl Selector for SupergraphSelector {
 
     fn on_response(&self, response: &supergraph::Response) -> Option<opentelemetry::Value> {
         match self {
+            SupergraphSelector::Query { query, .. } => {
```

Comment:
If it happens in `on_response` we might duplicate this behavior for `on_error` as even on error it could be interesting to have these selectors applied right ?

Reply:
It could be interesting. I've added it: 837aa9a7d3263d060f1e3f92aa2589c733b7442a

Reply:
@bnjjj One thing about that. I'm not clear how (or if) I should document this. Any suggestions?